internal/app/bindata.go

```diff
@@ -3,7 +3,7 @@
 // data/role.yaml
 // DO NOT EDIT!
 
-package main
```

Comment:
This file is no longer needed together with `data/role.yaml` as it was only needed for Tiller.